client/src/components/Nav/Category.tsx

```diff
@@ -0,0 +1,316 @@
+                onClick={parentClick}
+              >
+                <div className="parent-item">
+                  <p>{category.name}</p>
+                  <i className="fas fa-chevron-down"></i>
+                </div>
+                <ul className={'child ' + (category.children ? '' : 'none')} onMouseEnter={childHover}>
+                  <li className="all" key={parentActive} data-id={parentActive} onClick={childClick}>
+                    전체보기
+                  </li>
+                  {category.children ? (
+                    category.children.map((childCategory) => (
+                      <li key={childCategory.id} data-id={childCategory.id} onClick={childClick}>
+                        {childCategory.name}
+                      </li>
+                    ))
+                  ) : (
+                    <></>
+                  )}
+                </ul>
+              </li>
+            );
+          })}
+        </ul>
+      </CategoryMenu>
+    </CategoryWrapper>
+  );
+};
+
+const CategoryWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+  > svg {
+    width: 2rem;
+    height: 2rem;
+    margin-right: 0.3em;
+    path {
+      stroke: black;
+      stroke-width: 2.5;
+      stroke-linecap: round;
+      stroke-linejoin: round;
+    }
+  }
+  > p {
+    height: 100%;
+    text-align: center;
+  }
+  @media (max-width: 568px) {
+    position: static;
+    > p {
+      display: none;
+    }
+  }
+`;
+const CategoryMenu = styled.div`
+  @keyframes cate-fade-in {
+    from {
+      transform: translateX(-100%);
+    }
+    to {
+      transform: translateX(0);
+    }
+  }
+  @keyframes cate-fade-out {
+    to {
+      transform: translateX(-100%);
+    }
+    from {
+      transform: translateX(0);
+    }
+  }
+  position: absolute;
+  top: 100%;
+  left: 0;
+  font-size: 1rem;
+  > .header {
+    display: none;
+  }
+  ul {
+    width: 200px;
+    border-radius: 6px;
+    color: white;
+    > li {
+      padding: 1em;
+    }
+  }
+  > .parent {
+    background-color: ${({ theme }) => theme.color.baeminPrimary};
+    > li {
+      .parent-item {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        &:hover {
+          cursor: pointer;
+        }
+        i {
+          display: none;
+        }
+      }
+      > .child {
+        display: none;
+        position: absolute;
+        height: 100%;
+        left: 100%;
+        top: 0;
+      }
+    }
+    > li.active {
+      background-color: ${({ theme }) => theme.color.baeminDark};
+      > .child:not(.none) {
+        display: block;
+        background-color: ${({ theme }) => theme.color.baeminDark};
+        > .all {
+          display: none;
+        }
+        > li:hover {
+          cursor: pointer;
+          text-decoration: underline;
+          text-underline-position: under;
+        }
+      }
+    }
+  }
+  // mobile ui
+  @media (max-width: 568px) {
```

Comment:
```suggestion
  @media (max-width: ${({ theme }) => theme.media.mobile}) {
```

`DefaultTheme`에 화면 크기를 지정해 놔서 사용하면 좋지 않을까 합니다.

Reply:
좋습니다~~~